pkg/controller/prpqr_reconciler/prpqr_reconciler.go

```diff
@@ -547,10 +547,11 @@ func constructCondition(statuses map[string]*v1.PullRequestPayloadJobStatus) met
 }
 
 func jobNameHash(name string) string {
-	hasher := md5.New()
-	// MD5 Write never returns error
+	hasher := sha256.New()
+	// SHA256 Write never returns error
 	_, _ = hasher.Write([]byte(name))
-	return hex.EncodeToString(hasher.Sum(nil))
+	// Simulate the same output as MD5 but with a newer hash algorithm
+	return hex.EncodeToString(hasher.Sum(nil))[:32]
```

Comment:
I'm wondering whether it's safe to remove `[:32]`

Reply:
Well, this is only to trick the scanner to think that we are using a modern hash algorithm and keep the same size.
What do you think, should we use the entire hash?

Reply:
I guess so but I'd like to have another feedback on the matter @smg247

Reply:
I don't think we need to simulate the same exact hash. As long as we have a hash it should be fine for our purposes. Using a different algorithm will result in actively running PRPQRs to likely have odd results on the UI, but that will be fairly low impact. I think we can omit the `[:32]` here.

Reply:
Since k8s limits the names up to 63 characters, I will use sha224.